Exercise_4/docs/part-1.md

```diff
@@ -1,122 +1,306 @@
+**When dealing with EHR data you will likely encounter two types: structured and unstructured.**
 
-Structured data, in-contrast, is, as the name suggests, restricted to heavily formatted data-types and structures. This could include text fields like name, regular-expressions, like phone number, and alpha-numeric coding terminology.
+* Unstructured data, typically taking the form of encounter notes, audio, video, or image files, has its own set of significant challenges, and will be out of scope for this tutorial.
+* Structured data, in-contrast, is, as the name suggests, restricted to heavily formatted data types and structures. This could include text fields like name, regular-expressions, like phone number, and alpha-numeric coding terminology. 
 
-We'll look at a handful of specific coding standards relevant to interoperability. Specifically:
-> * LOINC 
-* CPT
-* SNOMED
-* ICD-10
-* RxNorm
+Commonly used coding standards for structured data include:  **LOINC, CPT, SNOMED CT, ICD-10, and RxNorm, etc.**​ It is important to have a general understanding of these coding systems because they have become the standard for rendering structured data in an interoperable manner.
 
 ### LOINC
 
-The **Logical Observation Identifiers Names and Codes (LOINC®)**, is a proprietary clinical terminology that is important for laboratory test orders and results, and is one of a suite of designated standards for use in U.S. Federal Government systems for the electronic exchange of clinical health information.
+**Logical Observation Identifiers Names and Codes (LOINC®)**, is a proprietary clinical terminology that is important for laboratory test orders and results, and is one of a suite of designated standards for use in U.S. Federal Government systems for the electronic exchange of clinical health information. 
+
+Managed by the Regenstrief Institute, LOINC has been identified by the Health Level 7 (HL7) Standards Development Organization as a preferred diagnostic report code set for laboratory test identification in transactions between health care facilities, laboratories, laboratory testing devices and public health authorities.
 
-Managed by the Regenstrief Institute, LOINC has been identified by the HL7 Standards Development Organization as a preferred code set for laboratory test identification in transactions between health care facilities, laboratories, laboratory testing devices and public health authorities.
+See here: [https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code](https://www.hl7.org/fhir/diagnosticreport-definitions.html#DiagnosticReport.code){target=_blank}
 
-LOINC codes are dvidied into Test Order and Test Result codes.
 
-??? abstract "Test Order Code Structure and Examples"
+LOINC codes are divided into Test Order and Test Result codes.
```

Comment:
```suggestion
LOINC codes are divided into Order and Observation codes.
```
"Test Order" and "Test Result" is from Mayo Clinic, and that is not the terminology used by LOINC itself.